tutorial/creating_packages/configure_options_settings.rst

```diff
@@ -0,0 +1,269 @@
+Configure settings and options in recipes
+=========================================
+
+.. important::
+
+    In this example, we retrieve the CMake Conan package from a Conan repository with
```

Comment:
It seems this is no longer true? This is not requiring the ``cmake`` conan package, or maybe it is just not seen in excerpts, but it is there, as continuation of previous exercises?

Reply:
Yes, It does not require CMake but fmt, I change CMake for fmt 👍